.clj-kondo/config.edn

```diff
@@ -345,45 +345,49 @@
+  metabase.db.liquibase/with-liquibase                                  clojure.core/let
+  metabase.driver.mongo.query-processor/mongo-let                       clojure.core/let
+  metabase.driver.mongo.util/with-mongo-connection                      clojure.core/let
+  metabase.driver.sql-jdbc.actions/with-jdbc-transaction                clojure.core/let
+  metabase.driver.sql-jdbc.execute.diagnostic/capturing-diagnostic-info clojure.core/fn
+  metabase.integrations.ldap/with-ldap-connection                       clojure.core/fn
+  metabase.mbql.schema.macros/defclause                                 clj-kondo.lint-as/def-catch-all
+  metabase.models.setting.multi-setting/define-multi-setting            clojure.core/def
+  metabase.models.setting/defsetting                                    clj-kondo.lint-as/def-catch-all
+  metabase.public-settings.premium-features/defenterprise               clj-kondo.lint-as/def-catch-all
+  metabase.public-settings.premium-features/defenterprise-schema        clj-kondo.lint-as/def-catch-all
+  metabase.public-settings.premium-features/define-premium-feature      clojure.core/def
+  metabase.query-processor.error-type/deferror                          clojure.core/def
+  metabase.sync.util/sum-for                                            clojure.core/for
+  metabase.sync.util/with-emoji-progress-bar                            clojure.core/let
+  metabase.test.data.interface/defdataset                               clojure.core/def
+  metabase.test.data.interface/defdataset-edn                           clojure.core/def
+  metabase.test/are+                                                    clojure.test/are
+  metabase.test/defdataset                                              clojure.core/def
+  metabase.test/with-open-channels                                      clojure.core/let
+  metabase.test/with-temp-file                                          clojure.core/let
+  metabase.test/with-user-in-groups                                     clojure.core/let
+  metabase.util.files/with-open-path-to-resource                        clojure.core/let
+  metabase.util.ssh/with-ssh-tunnel                                     clojure.core/let
+
+  clojure.core.logic/defne           clj-kondo.lint-as/def-catch-all
+  clojurewerkz.quartzite.jobs/defjob clojure.core/defn
+  honeysql.helpers/defhelper         clj-kondo.lint-as/def-catch-all
+  honeysql.util/defalias             clojure.core/def
+  monger.operators/defoperator       clojure.core/def
+  potemkin.types/defprotocol+        clojure.core/defprotocol
+  potemkin.types/defrecord+          clojure.core/defrecord
+  potemkin.types/deftype+            clojure.core/deftype
+  potemkin/defprotocol+              clojure.core/defprotocol
+  potemkin/defrecord+                clojure.core/defrecord
+  potemkin/deftype+                  clojure.core/deftype
+  toucan.db/with-call-counting       clojure.core/fn}
```

Comment:
I added `metabase.driver.sql-jdbc.actions/with-jdbc-transaction`, but these were out of order, so I didn't know where to add it. Also it looks like stuff was divided up into two groups -- Metabase stuff and 3rd-party stuff -- but `toucan.db/with-call-counting` was in the Metabase stuff group. Made my eyes burn. Fixed it and indented it nicely.